src/components/Users/UsersList.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useUsersContext } from '../../hooks/useUsersContext';
+import { User } from '../../types';
+
+type Props = {
+  onCloseDropdown: () => void;
+};
+
+export const UsersList: FC<Props> = ({ onCloseDropdown }) => {
+  const { onSelectUser, selectedUser, users } = useUsersContext();
+
+  const handleSelectUser = (user: User) => {
+    onSelectUser(user);
+    onCloseDropdown();
+  };
+
+  return (
+    <>
+      {users.map((user) => (
+        <a
+          href={`#user-${user.id}`}
```

Comment:
Using anchor tags with a href attribute that starts with '#' causes the page to jump. If you don't want this behavior, you can use a button or div element instead.